app/templates/partials/summary/summary.html

```diff
@@ -4,10 +4,11 @@
   <div class="summary">
 
     {% for group in content.summary.groups %}
+      {% if group.title and content.summary.summary_type == 'summary' %}
```

Comment:
Based on the design it should only show the section titles on the section summaries
https://trello-attachments.s3.amazonaws.com/56d9b1dcc08c137b23e86234/5a69ed2da9df0bb13799f868/1038x671/c13873dbe5bba999ab052ceab568caea/Screen_Shot_2018-01-22_at_14.26.25.png

Reply:
I mean, that images doesn't explicitly say that, it could just be that if there is only 1 group in a section then don't show the title?
Confirm this behaviour with Will